### PR TITLE
chore: examples, add subscription examples

### PR DESCRIPTION
**Pull request by YuriBuerov**
#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: examples/graphql-sse-subscriptions/.wundergraph/wundergraph.config.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+type Subscription {
+  greetings: String
+}
+`;
+
+const greetings = introspect.graphql({
+	apiNamespace: 'sse',
+	url: 'http://127.0.0.1:4000/graphql/stream',
+	loadSchemaFromString: schema,
+	subscriptionsUseSSE: true,
```
Comment (StarpTech) [inline examples/graphql-sse-subscriptions/.wundergraph/wundergraph.config.ts:31]: Wasnt it?

```ts
subscription: {
  useSSE: true
}
```

File: examples/graphql-sse-subscriptions/.wundergraph/wundergraph.server.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { configureWunderGraphServer } from '@wundergraph/sdk';
+import type { HooksConfig } from './generated/wundergraph.hooks';
+import type { InternalClient } from './generated/wundergraph.internal.client';
+
+export default configureWunderGraphServer<HooksConfig, InternalClient>(() => ({
+	hooks: {
+		queries: {},
+		mutations: {},
+	},
+	graphqlServers: [
```
Comment (StarpTech) [inline examples/graphql-sse-subscriptions/.wundergraph/wundergraph.server.ts:11]: Remove things that aren't necessary for the example. That makes our life easier to maintain it.

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: examples/graphql-apollo-subscripptions/.wundergraph/wundergraph.config.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+						// change this before deploying to production to the actual domain where you're deploying your app
+						'http://localhost:3000',
+				  ]
+				: ['http://localhost:3000', new EnvironmentVariable('WG_ALLOWED_ORIGIN')],
+	},
+	dotGraphQLConfig: {
+		hasDotWunderGraphDirectory: false,
+	},
+	security: {
+		enableGraphQLEndpoint: process.env.NODE_ENV !== 'production' || process.env.GITPOD_WORKSPACE_ID !== undefined,
```
[the remark this answers is not in the document]
Comment (YuriBuerov) [reply examples/graphql-apollo-subscripptions/.wundergraph/wundergraph.config.ts:51]: yeah, missed it

File: examples/graphql-sse-subscriptions/.wundergraph/wundergraph.config.ts (hunk 1 above)
Comment (YuriBuerov) [reply examples/graphql-sse-subscriptions/.wundergraph/wundergraph.config.ts:31]: No, it wasn't. 
```
subscriptionsUseSSE
subscriptionsUrl
```

**Comment by StarpTech** [comment]
Whats missing here?

**Comment by jensneuse** [comment]
> Whats missing here?

non failing tests

**Comment by StarpTech** [comment]
@YuriBuerov, how can we move forward?

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**